lib/utils/matrix_sdk_extensions/flutter_hive_collections_database.dart

```diff
@@ -73,6 +77,15 @@ class FlutterHiveCollectionsDatabase extends HiveCollectionsDatabase {
       'hive_collections_${client.clientName.replaceAll(' ', '_').toLowerCase()}',
       await _findDatabasePath(client),
       key: hiverCipher,
+      startMigrationProcess: (currentVersion, newVersion) async {
+        Logs().d(
+          'FlutterHiveCollectionsDatabase::startMigrationProcess() Starting migration process',
+        );
+        Logs().d(
+          'FlutterHiveCollectionsDatabase::startMigrationProcess() CurrentVersion - $currentVersion || NewVersion - $newVersion',
+        );
+        canMigrateToMDatabase = true;
```

Comment:
```
abstract class MigrateSteps {
   Future<void> onMigrate(oldVersion, newVersion) {
   }
}
```

```
class Migrate6to7 extends MigrateSteps {
   //override
}
```

```
Migrate6to7().onMigrate(current, newVersion);
```

Reply:
Dialog
Title: Upgrade App
Message: `Twake Chat has a new version ready for you. Upgrade now for enhanced features and a smoother experience.`
Button: Ok